inventories/seapath_basic_cluster_definition_example.yml

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
In your commit message: exemple --> example

Reply:
Done